tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -7342,3 +7344,329 @@ def test_scorer_operations(store: SqlAlchemyStore):
     # Test list_scorer_versions for non-existent scorer
     with pytest.raises(MlflowException, match="Scorer with name 'non_existent_scorer' not found"):
         store.list_scorer_versions(experiment_id, "non_existent_scorer")
+
+
+def _create_experiments(store, name):
+    return store.create_experiment(name)
+
+
+def _create_trace_for_correlation(store, experiment_id, spans=None, assessments=None, tags=None):
```

Comment:
Nit: Any way we can shorten / simplify this utility? (currently quite large)

Reply:
understatement of the millenium. That's what I get for adding in "OTeL compliant trace". Opus done lost its mind.